README.md

```diff
@@ -1,21 +1,51 @@
 ## Matrix.org Website
 
-If you want to contribute to the website, make sure the problem you're trying to
-fix or the feature you want to implement has been discussed in our issue tracker
-and that we are interested in reviewing and merging such a contribution.
+# Table of Contents
+- [Overview](#overview)
+- [Building the Website](#building-the-website)
+- [Contribution Guidelines](#contribution-guidelines)
+- [Contact Us](#contact-us)
 
-To discuss maintenance of this site, please come talk to the team in
-[#matrix.org-website:matrix.org](https://matrix.to/#/#matrix.org-website:matrix.org).
+# Overview
+
+Matrix.org is a decentralized, open standard for real-time communication which allows users to collaborate in public and private rooms over IP. Matrix's purpose is to allow users to text and call without thinking about what app or phone the other user has. 
 
-## Building the website
 
-Install the latest version of [Zola](https://www.getzola.org), clone the repo, and from the root of the
-repo type
 
+## Building the website
+To contribute or run the website locally, follow the instructions below:
+1. '''
+git clone https://github.com/matrix-org/matrix.org.git
+cd matrix.org
```

Comment:
This seems to not be working with Github Markdown. Could you fix that please? Have a look at the rich diff button on the top right of this file to see how it is being rendered